client/src/services/resourceService.test.js

```diff
@@ -11,13 +11,13 @@ describe("ResourceService", () => {
 		it("sends an appropriate GET request", async () => {
 			let request;
 			server.use(
-				rest.get("/api/resources", (req, res, ctx) => {
-					request = req;
+				rest.get("/api/resources", async (req, res, ctx) => {
+					request = await req;
 					return res(ctx.json({ resources: [] }));
 				})
 			);
 			await service.getDrafts();
-			expect(request.url.searchParams.get("draft")).toBe("true");
+			await expect(request.url.searchParams.get("draft")).toBe("true");
```

Comment:
❓ Not sure why these changes have been made. Neither `req` nor `expect(...).toBe(...)` is a promise. Did this test fail or timeout at some point?